src/Stoa.ts

```diff
@@ -127,6 +127,7 @@ class Stoa extends WebService
         this.app.get("/utxo/:address", this.getUTXO.bind(this));
         this.app.get("/wallet/transactions/history/:address", this.getWalletTransactionsHistory.bind(this));
         this.app.get("/wallet/transaction/overview/:hash", this.getWalletTransactionOverview.bind(this));
+        this.app.get("/pending_transactions/:address", this.getPendingTransactions.bind(this));
```

Comment:
I recommend you to choose one of the two below.
```suggestion
        this.app.get("/wallet/transactions/pending/:address", this.getWalletTransactionsPending.bind(this));
```

```suggestion
        this.app.get("/transactions/pending/:address", this.getTransactionsPending.bind(this));
```

Reply:
You'd better do that.